cmd/main.go

```diff
@@ -67,7 +66,7 @@ func processGeneratorConfig(filePath string) []byte {
 		errorAndExit("failed to read file '%s': %s", filePath, err)
 	}
 
-	err = p.Config(fileData, path.Dir(cwd))
+	err = p.Config(fileData, cwd)
```

Comment:
I'm not sure why we stripped out the last bit of the path (which is the actual working directory). But it was preventing identifying when a `kustomization.yaml` pointed back to itself: https://github.com/stolostron/policy-generator-plugin/pull/65/commits/c0158ef76fbe2c1c94a503e2506869b9eb9b849d

@mprahl Do you recall why we did it this way? The tests pass without it, which is also confusing.

Reply:
I don't remember but I don't see a reason why it's like that. So I'm +1 to your change.